crates/bevy_ecs/src/event.rs

```diff
@@ -798,48 +810,77 @@ impl<'a, E: Event> ExactSizeIterator for EventIteratorWithId<'a, E> {
     }
 }
 
+/// A registry of all of the [`Events`] in the [`World`], used by [`event_update_system`]
+/// to update all of the events.
 #[doc(hidden)]
```

Comment:
I don't think this should be [doc(hidden)]: it's required for use in raw bevy_ecs.

Reply:
Done.